src/crimson/tools/perf_staged_fltree.cc

```diff
@@ -13,6 +13,7 @@
 #include "crimson/os/seastore/onode_manager/staged-fltree/node_extent_manager.h"
 
 #include "test/crimson/seastore/onode_tree/test_value.h"
+#include "test/crimson/gtest_seastar.h"
```

Comment:
Looks `perf_staged_fltree` doesn't directly depends on this header, should be better to move it into `transaction_manager_test_state.h`.